documentation/how-to/tracking.en.md

```diff
@@ -4,32 +4,65 @@ title: Tracking
 
 # Tracking
 
-You have the possibility to do tracking on lines and polygons in QField.
+QField allows you to create points, lines, and polygons from tracking your position.
+
+## Activate tracking
 
-## Usage
 :material-tablet-android:{ .device-icon } Fieldwork
 
 ![type:video](https://player.vimeo.com/video/499958163)
 
-It's available when GNSS is enabled. A tracking session can be started
-in the legend by long-pressing on the layer. The time interval and the
-minimum distance can be entered individually to define when a vertex
-needs to be set.
+To activate tracking, you must first make sure the positioning is active. Then,
+open the side dashboard, long-press on a legend layer within which you want to
+save your tracks to, and select the *Setup tracking* button to configure
+the tracking session.
+
+!![](../assets/images/tracking-layer-properties.png)
+
+Two constraints are available to limit the number of vertices recorded
+during tracing:
+- A minimum time interval; and
+- A minimum distance.
+
+!![](../assets/images/tracking-settings.png)
+
+Once you have finished configuring your tracing session, hit the *Start tracking*
+button to begin recording. At this stage, a feature form will appear, which allows
+you to define the attributes of the feature(s) that will be created while QField
+tracks your position.
+
+For line and polygon layers, a single feature - its geometry formed of vertices
+from the recorded positions - will be create per tracking session.  For point
+layers, a feature will be created for each recorded position, with attribute
+values entered in the form remembered across features.
+
+!!! note
+    QField will skip the feature form step if the layer is configured to hide all attributes. This can streamline the process of starting tracking sessions.
```

Comment:
... or if the form has been configured to be hidden in the attribute form settings

![image](https://user-images.githubusercontent.com/588407/187025845-bd2d86c3-ea46-4e12-951e-3302207d3572.png)

Reply:
@m-kuhn , we'll have to make this true by the time we publish 2.3 :)

Reply:
It's already respected for simple feature creation IIRC

Reply:
@m-kuhn , it is now a truth: https://github.com/opengisch/QField/pull/3247/commits/e1e98bb94bb034f48f6850b32865c785485ac9a4 :)